tests/databases/value/fixtures/mongo_db.py

```diff
@@ -1,26 +1,30 @@
 import pytest
 import stk
+import pymongo
 
 from ..case_data import CaseData
-from ...utilities import MockMongoClient
 
 
 @pytest.fixture(
     params=(
         CaseData(
             database=stk.ValueMongoDb(
-                mongo_client=MockMongoClient(),
+                mongo_client=pymongo.MongoClient(),
                 collection='values',
-                lru_cache_size=0,
+                database='_test_database',
```

Comment:
Out of interest, why use the underscore here?

Reply:
Just to make clear its not for regular use & make it unlikely that the test database will conflict with an actual legitimate database in-use by a user. A random string might be in order for that too, but I figured this was fine for now.